eos-phone-home

```diff
@@ -171,9 +172,9 @@ class PhoneHome(object):
             uploading_enabled = config['global'].getboolean('uploading_enabled',
                 uploading_enabled)
             environment = config['global'].get('environment', environment)
-        except Exception as e:
-            print("Unable to read %s, assuming fallback values." %
-                (permissions_file, ), e, file=sys.stderr)
+        except:
+            log.info("Unable to read %s, assuming fallback values.",
+                     permissions_file)
```

Comment:
Ditto: We lose the exception/traceback for the exception with this change I think.